CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 ## Contributing
 
-[Contributions](//github.com/visjs/vis-charts/blob/master/misc/how_to_help.md) to the vis.js library are very welcome! [We can't do this alone](//github.com/visjs/vis-charts/blob/master/misc/we_need_help.md).
+Contributions to the vis.js library are very welcome! We can't do this alone.
```

Comment:
```suggestion
[Contributions](https://github.com/visjs/vis-timeline/blob/master/CONTRIBUTING.md) to the vis.js library are very welcome! We can't do this alone.
```

Reply:
It's a link to the same file. I don't see a reason to do this